book/src/developers/designs/auth.md

```diff
@@ -0,0 +1,428 @@
+# Authentication Use Cases
+
```

Comment:
We might consider adding a header that this is potentially outdated now since this was true at the time but I think some of the ideas here have evolved.

Reply:
Yeah, this was me just generally going through and doing a first-pass formatting thing, then giving up after realising it needs a full rewrite 😄

Reply:
It's not linked in the book, FWIW